examples/api-samples/src/electron-browser/updater/sample-updater-frontend-contribution.ts

```diff
@@ -68,18 +67,6 @@ export namespace SampleUpdaterMenu {
     export const MENU_PATH: MenuPath = [...CommonMenus.FILE_SETTINGS_SUBMENU, '3_settings_submenu_update'];
 }
 
-@injectable()
-export class SampleUpdaterClientImpl implements SampleUpdaterClient {
-
-    protected readonly onReadyToInstallEmitter = new Emitter<void>();
-    readonly onReadyToInstall = this.onReadyToInstallEmitter.event;
-
-    notifyReadyToInstall(): void {
-        this.onReadyToInstallEmitter.fire();
-    }
-
-}
```

Comment:
This might be the most important change with the new RPC/proxy API: we no longer setup "clients" to get notifications back from the remote: we can use `Event` fields directly instead.

Reply:
I'm not sure this is actually an improvement. How is this better than the old way of setting up service objects on both ends?

Reply:
I fear this might make initialization of service objects more complicated.

Reply:
I don't see how this makes things more complicated. I keep deleting code to use this new API.